test cases/common/21 global arg/meson.build

```diff
@@ -1,4 +1,14 @@
-project('global arg test', 'cpp', 'c')
+project('global arg test', 'c')
+
+if add_languages('cpp', required: false)
+  cpp = meson.get_compiler('cpp')
+else
+  error('MESON_SKIP_TEST No C++ compiler available.')
+endif
+
+if host_machine.system() == 'windows' and fcid == 'pgi'
+  error('MESON_SKIP_TEST: PGI on Windows does not have a C++ compiler')
+endif
```

Comment:
Same here, skip the C++ part if there is no c++ compiler, not the whole test.

Reply:
resolved by 0ea9323c